.github/workflows/npm-publish.yml

```
@@ -0,0 +1,42 @@
+name: "🚀 publish"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    name: 🚀 NPM publish
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: ☄️ checkout
+        uses: actions/checkout@v2
+
+      - name: 🐱‍🏍 node
+        uses: actions/setup-node@v1
```

Not sure if this is from another repos Workflow, but there is a much newer version of this action
```suggestion
        uses: actions/setup-node@v3
```